src/main/java/io/confluent/copycat/jdbc/DataConverter.java

```diff
@@ -67,48 +68,45 @@ public static GenericRecord convertRecord(Schema schema, ResultSet resultSet)
         log.warn("Ignoring record due to SQL error:", e);
       }
     }
-    return builder.build();
+    return struct;
   }
 
 
   private static void addFieldSchema(ResultSetMetaData metadata, int col,
-                                     SchemaBuilder.FieldAssembler<Schema> fields)
+                                     SchemaBuilder builder)
+                                     // SchemaBuilder.FieldAssembler<Schema> fields)
       throws SQLException {
     // Label is what the query requested the column name be using an "AS" clause, name is the
     // original
     String label = metadata.getColumnLabel(col);
     String name = metadata.getColumnName(col);
     String fieldName = label != null && !label.isEmpty() ? label : name;
-    // Create the field for each type case to handle
-    SchemaBuilder.FieldBuilder<Schema> builder = fields.name(fieldName);
+
     // FIXME builder.aliases() or builder.doc()?
```

Comment:
Is there any info in the column metadata that could be used to fill out any of the other schema fields here? Or should we just remove this comment? Things like `doc` probably can't be filled in, but what about default value?